sdk/core/core/test/cmocka/main.c

```diff
@@ -1,59 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
-#include "az_test_definitions.h"
+#include <stdlib.h>
 
 #include <setjmp.h>
 #include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
 
 #include <cmocka.h>
 
-#include <az_json.h>
+#include "az_test_definitions.h"
 
 #include <_az_cfg.h>
 
-/**
- * How to add tests:
```

Comment:
Can we update these instructions that tell us how to add new tests instead of removing them?

Reply:
should be in contributing guide and not in the code itself.  (I got that feedback time ago about not adding docs into code directly XD)